blocked-edges/4.7.24.yaml

```diff
@@ -0,0 +1,3 @@
+to: 4.7.24
+from: 4\.7\..*
```

Comment:
/hold

I don't understand why 4.6 -> 4.7.24 is ok.

Reply:
Yeah we need to block that too, sorry I forgot about that when proposing this. So everything to 4.7.24